docs/user_guide.md

```diff
@@ -556,76 +531,184 @@ If you are using a self-signed cert, make sure to copy the CA cert into ```/etc/
+It might be necessary to connect Harbor to other scanners for corporate compliance reasons, or because your organization already uses a particular scanner. Different scanners also use different vulnerability databases, capture different CVE sets, and apply different severity thresholds. By connecting Harbor to more than one vulnerability scanner, you broaden the scope of your protection against vulnerabilities.
 
-**Vulnerability metadata** 
+**IMPORTANT**: Clair is an optional component. To be able to use Clair or to configure additional scanners, you must have enabled Clair when you installed your Harbor instance. You configure additional scanners in the Harbor interface, after you have installed Harbor. 
 
-Clair depends on the vulnerability metadata to complete the analysis process. After the first initial installation, Clair will automatically start to update the metadata database from different vulnerability repositories. The updating process may take a while based on the data size and network connection. If the database has not been fully populated, there is a warning message at the footer of the repository datagrid view.
-![browse project](img/clair_not_ready.png)
+- For information about installing Harbor with Clair, see the [Installation and Configuration Guide](installation_guide.md). 
+- For information about adding additional scanners, see [Connect Harbor to Additional Vulnerability Scanners](#pluggable-scanners) below.
 
-The 'database not fully ready' warning message is also displayed in the **'Vulnerability'** tab of **'Configuration'** section under **'Administration'** for your awareness.
-![browse project](img/clair_not_ready2.png)
+You can manually initiate scanning on a particular image, or on all images in Harbor. Additionally, you can also set a policy to automatically scan all of the images at specific intervals.
 
-Once the database is ready, an overall database updated timestamp will be shown in the **'Vulnerability'** tab of **'Configuration'** section under **'Administration'**. 
+<a id="pluggable-scanners"></a>
+### Connect Harbor to Additional Vulnerability Scanners
+
+To connect Harbor to additional vulnerability scanners, you must have enabled the default Clair scanner when you deployed Harbor. You must install and configure an instance of the additional scanner according to the scanner vendor's requirements. The scanner must expose an API endpoint to Harbor that permits image pushes and pulls. You can deploy multiple different scanners, and multiple instances of the same type of scanner.
+
+1. Log in to the Harbor interface with an account that has Harbor administrator privileges.
+1. Expand **Administration**, and select **Interrogation Services**. 
+   ![Interrogation Services](img/interrogation-services.png)
+1. Click the **New Scanner** button.
+1. Enter the information to identify the scanner.
+   - A name for this scanner instance, to display in the Harbor interface.
```

Comment:
>A name for this scanner instance, to display in the Harbor interface.

The name is unique

Reply:
Added in https://github.com/goharbor/harbor/pull/10029/commits/a1eac99a49aa192cb5048c5eaa8a4e2a756cd134.